### ISSUE TITLE
Add documentation on how to develop renin locally

### ISSUE THREAD

**Issue by stianjensen** [ninjadev/renin#72]
I'm having some trouble figuring out exactly how to run renin locally to develop the UI while running it with the example-project. It would be nice with a small section describing that process in the README or somewhere appropriate.

I've tried running `yarn` in the example-project, and inside the example-project inside the lib-folder inside renin, with no success.

**Comment by sigvef** [comment]
Yeah, there isn't any great setup for that right now :/ Before release of the cli (for npx renin init), renin and the example project was all in one big project, so there was easy default live reloading for everything. Now, since renin is only a library project, and the example project is a separate thing, then there is no real nice way to work on them together at the moment.

One workaround is to change the renin install path in example project to `"file:../renin"`, and then rebuild renin and reinstall node_modules of the example project. That way, we can at least run example project with whatever changes we make to the renin project.

A better approach though, would be to restructure the projects a little bit so that we both get a test bench to live-reload-develop renin locally, and also have a pristine example project that we can copy-paste into new projects spawned by `npx renin init`.
